src/components/ButtonPanel.js

```diff
@@ -4,34 +4,34 @@ import './ButtonPanel.css';
 
 function ButtonPanel() {
   return (
-    <div className="ButtonPanel" id="button-panel">
-      <div className="group1">
-        <Button buttonName="AC" />
-        <Button buttonName="+/-" />
-        <Button buttonName="%" />
+    <div className="button-panel" id="button-panel">
+      <div className="group">
+        <Button buttonName="AC" color="#dfdfdf" />
+        <Button buttonName="+/-" color="#dfdfdf" />
+        <Button buttonName="%" color="#dfdfdf" />
         <Button buttonName="÷" />
       </div>
-      <div className="group2">
-        <Button buttonName="7" />
-        <Button buttonName="8" />
-        <Button buttonName="9" />
+      <div className="group">
+        <Button buttonName="7" color="#dfdfdf" />
+        <Button buttonName="8" color="#dfdfdf" />
+        <Button buttonName="9" color="#dfdfdf" />
         <Button buttonName="x" />
       </div>
-      <div className="group3">
-        <Button buttonName="4" />
-        <Button buttonName="5" />
-        <Button buttonName="6" />
+      <div className="group">
+        <Button buttonName="4" color="#dfdfdf" />
+        <Button buttonName="5" color="#dfdfdf" />
+        <Button buttonName="6" color="#dfdfdf" />
         <Button buttonName="-" />
       </div>
-      <div className="group4">
-        <Button buttonName="1" />
-        <Button buttonName="2" />
-        <Button buttonName="3" />
+      <div className="group">
+        <Button buttonName="1" color="#dfdfdf" />
+        <Button buttonName="2" color="#dfdfdf" />
+        <Button buttonName="3" color="#dfdfdf" />
         <Button buttonName="+" />
       </div>
-      <div className="group5">
-        <Button buttonName="0" />
-        <Button buttonName="." />
+      <div className="group">
+        <Button buttonName="0" wide color="#dfdfdf" />
+        <Button buttonName="." color="#dfdfdf" />
         <Button buttonName="=" />
       </div>
```

Comment:
**OPTIONAL**: A lot of duplicated code 😞, do you think we could do this without repeating so much?